datalab/datalab_session/tasks.py

```diff
@@ -9,11 +9,15 @@
 log = logging.getLogger()
 log.setLevel(logging.INFO)
 
+
+TIME_LIMIT = 60 * 60 * 1000  # 1 hour time limit in ms
+
+
 # Retry network connection errors 3 times, all other exceptions are not retried
 def should_retry(retries_so_far, exception):
     return retries_so_far < 3 and isinstance(exception, RequestException)
 
-@dramatiq.actor(retry_when=should_retry)
+@dramatiq.actor(retry_when=should_retry, time_limit=TIME_LIMIT)
```

Comment:
Default time limit is 10 minutes - increasing to 1 hour since I hit the time limit doing a 20 image normalization locally. We could make it a little higher but eventually we want hung operations to time out and fail so it doesn't block stuff forever...